span_test.go

```diff
@@ -3,6 +3,7 @@ package basictracer
 import (
 	"testing"
 
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
Ah, it's not needed, but this is the style for this import throughout the rest of basictracer-go.  I can do whichever you prefer.